pages/videos.tsx

```diff
@@ -0,0 +1,111 @@
+import PopupVideo from '@components/Video/PopupVideo'
+import VideoCard from '@components/Video/VideoCard'
+import { Fragment, useState } from 'react'
+import { Dialog, Transition } from '@headlessui/react'
+
+interface Video {
+  youtubeVideoID: string
+  title: string
+}
+
+interface VideosGridProps {
+  videos: Array<Video>
+}
+
+const VideosGridView: React.FC<VideosGridProps> = ({ videos }) => {
+  videos = [
+    {
+      youtubeVideoID: 's9F8pu5KfyM',
+      title: 'Why Do Computers Suck At Math?'
+    },
+    {
+      youtubeVideoID: 'B1t4Fjlomi8',
+      title: 'Why do computers use RGB for colours, and not RBY?'
+    },
+    {
+      youtubeVideoID: 'njdJeu95p6s',
+      title: 'Top 3 Ways to Center a DIV with CSS'
+    },
+    {
+      youtubeVideoID: 'njdJeu95p6s',
+      title: 'Top 3 Ways to Center a DIV with CSS'
+    }
+  ]
```

Comment:
We are overriding the videos prop here we should either keep `videos` a local variable or a prop not both